website/content/docs/commands/plugin/info.mdx

```diff
@@ -9,6 +9,12 @@ description: The "plugin info" command displays information about a plugin in th
 The `plugin info` displays information about a plugin in the catalog.
 The plugin's type of "auth", "database", or "secret" must be included.
 
+## deprecation_status field
+
+As of 1.12, all builtin plugins will have an associated Deprecation
+Status. This status will be reflected in the `deprecation_status` key/value
```

Comment:
I might have missed it but should we list the possible values for the `deprecation_status` key?

Reply:
Good idea! Since `Deprecation Status` is used in a few other locations: (auth and secrets), I think I'll create a new reference that can be leveraged by all. Interestingly, [the feature deprecation page](https://developer.hashicorp.com/vault/docs/deprecation) does not actually list the stages of deprecation. I'll go ahead and add some updates there as well.

Reply:
I'm going to go ahead and do this as a follow-up PR.